appveyor.yml

```diff
@@ -19,27 +19,20 @@ environment:
     - TOOLCHAIN: "nmake-vs-12-2013"
       PROJECT_DIR: examples\benchmark
 
-    - TOOLCHAIN: "vs-10-2010"
```

Comment:
As you can see [here](https://ci.appveyor.com/project/google/benchmark/branch/master) google/benchmark does not support vs 10 2012. But if you prefer, I'll provide links to the broken build.